src/java/org/apache/cassandra/security/FileBasedSslContextFactory.java

```diff
@@ -20,6 +20,7 @@
 
 import java.io.InputStream;
 import java.nio.file.Files;
+import java.nio.file.Paths;
```

Comment:
@maulin-vasavada you are using illegal imports, you should use `org.apache.cassandra.io.util.File.getPath` instead

````
checkstyle:
    [mkdir] Created dir: /home/fermat/dev/cassandra/cassandra-instaclustr/cassandra/build/checkstyle
[checkstyle] Running Checkstyle 8.45.1 on 2201 files
[checkstyle] [ERROR] /home/fermat/dev/cassandra/cassandra-instaclustr/cassandra/src/java/org/apache/cassandra/security/FileBasedSslContextFactory.java:23:1: Illegal import - java.nio.file.Paths. [IllegalImport]
````

Reply:
Sure, made the change. Thanks. Will figure out why I am unable to see the same checkstyle error locally.